tap_google_sheets/sync.py

```diff
@@ -388,6 +397,8 @@ def sync(client, config, catalog, state):
     LOGGER.info('last_datetime = {}, this_datetime = {}'.format(last_datetime, this_datetime))
     if this_datetime <= last_datetime:
         LOGGER.info('this_datetime <= last_datetime, FILE NOT CHANGED. EXITING.')
+        # Update file_metadata bookmark
+        write_bookmark(state, 'file_metadata', strftime(this_datetime))
```

Comment:
why do we need to update the state here? if the current bookmark is older than the state bookmark were reseting the state to the new bookmark. idk when this would really happen but the next run might re-sync data that we previously synced then

Reply:
Another change from Master